cover.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/sh
+set -e #break script on non-zero exitcode from any command
+set -x #display command being executed
+CTEST_OUTPUT_ON_FAILURE=ON
+export CTEST_OUTPUT_ON_FAILURE
+
+wget http://ftp.de.debian.org/debian/pool/main/l/lcov/lcov_1.11.orig.tar.gz
```

Comment:
Blog post I linked mentioned that lcov from apt is old and didn't work fo him. I can check how it would work for us.